skaha/VERSION

```diff
@@ -1,4 +1,4 @@
 ## deployable containers have a semantic and build tag
 # semantic version tag: major.minor
 # build version tag: timestamp
-TAGS="0.9.0 $(date -u +"%Y%m%dT%H%M%S")"
+TAGS="0.9.1 $(date -u +"%Y%m%dT%H%M%S")"
```

Comment:
You'll have to make this 0.9.2--I released a bug fix earlier.

Reply:
Done.